deployer/README.md

```diff
@@ -499,27 +669,64 @@ create/delete/get/get-all/update/ CILogon clients using the 2i2c administrative
   ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
   ```
 
-## Debugging helpers
+### The `exec` sub-command for executing shells and debugging commands
+
+This deployer `exec` sub-command can be used to
+exec a shell in various parts of the infrastructure.
+It can be used for poking around, or debugging issues.
 
-We also have some debug helpers commands that can be invoked as subcommands.
 These are primarily helpful for manual actions when debugging issues (during
 setup or an outage), or when taking down a hub. 
 
 All these commands take a cluster and hub name as parameters, and perform appropriate
 authentication before performing their function.
 
-### `component-logs`
+**Command line usage:**
+
+```bash
+ Usage: deployer exec [OPTIONS] COMMAND [ARGS]...                                                                       
+                                                                                                                        
+ Execute a shell in various parts of the infra. It can be used for poking around, or debugging issues.                  
+                                                                                                                        
+╭─ Options ────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ --help          Show this message and exit.                                                                          │
+╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Commands ───────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ debug                                                                                                                │
```

Comment:
Let's invert this, so `debug` is the top level, and `exec` is a sub-level? None of the things under `debug` are execing anything, while usually we `exec` things to debug something...

Reply:
So the layout would be:

```
- debug
  |- exec
  |- user-logs
  |- component-logs
  |- start-docker-proxy
```

Reply:
Alternatively, `debug` and `exec` can *both* be top level.